exams/pearson_test.py

```diff
@@ -1,32 +1,35 @@
 """
 Tests for exams.pearson module
 """
-from datetime import datetime
+from datetime import date, datetime
 from unittest import mock
 from unittest.mock import patch
 
 import io
```

Comment:
I usually group together the `import ...` and the `from ... import ... ` for the same groups. We do not have a preference about whether to put `import ...` first or `from ... import ... ` but I think we are pretty consistent about the grouping together without spaces.
Anyway, this is super minor nit